RELEASE_NOTES

```diff
@@ -3,6 +3,60 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.9.0 (September 23rd 2016)
+----------------------------
+
+feature release
+- note: vsc-base 2.5.4 or more recent is now required
+- various small enhancements, including:
+    - change option `--color` choices to auto/always/never (#1701, #1898, #1911)
+    - add support for 'hidden' easyconfig parameter (#1837)
+    - add support for using ccache and f90cache compiler caching tools (#1844, #1912)
+        - see --use-ccache and --use-f90cache
+    - update Cray metadata for 16.06 CrayPE release (#1851)
+    - also include patch files in --new-pr and --update-pr (#1852)
+    - handle deleted files in --new-pr (#1853)
+    - add support for --install-latest-eb-release (#1861)
+    - add support for hiding toolchains, see --hide-toolchains and 'hidden' key in 'toolchain' spec (#1871)
+    - add template for GitHub source URL (#1872)
+    - add support for combining --new-pr/--update-pr and --robot (#1881)
+    - add support for --list-software and --list-installed-software (#1883, #1910, #1917)
+    - print message on which extension is being installed, incl. progress counter (#1886, #1914)
+    - add support for --github-org to specify GitHub organisation rather than GitHub user (#1894)
+    - add support for running Lmod in debug mode (#1895)
+    - avoid needless use of deepcopy, speed up support for templating in easyconfigs (#1897)
+    - convert all_dependencies to a property in EasyConfig class (#1909)
+    - add support for --mpi-cmd-template (#1918)
+    - add support for --disable-mpi-tests (#1920)
+- various bug fixes, including:
+    - merge with develop when using --from-pr (#1838, #1867)
```

Comment:
didn't know that, good to read the Release Notes;

overall comment: all fine, lgtm